### PR TITLE
[zh] sync ist0134 and ist0143

### PR DESCRIPTION
**Pull request by windsonsea**
Add zh text to these 2 pages:

```
content/zh/docs/reference/config/analysis/ist0134/
content/zh/docs/reference/config/analysis/ist0143/
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by hanxiaop — APPROVED** [review]

File: content/zh/docs/reference/config/analysis/ist0134/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## 示例 {#example}
+
+您将收到以下消息：
+
+{{< text plain >}}
+Warning [IST0134] (ServiceEntry service-entry.default serviceentry.yaml:13) ServiceEntry addresses are required for this protocol.
+{{< /text >}}
+
+当集群的 ServiceEntry 未设置协议且缺少地址时：
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0134/index.md:18]: 这种配置中的字段 addresses 也需要翻译吗？

Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0134/index.md:18]: 貌似不用翻呐，英文没加 \`\`，没注意 😅

Comment (hanxiaop) [reply content/zh/docs/reference/config/analysis/ist0134/index.md:18]: 感觉可以给英文也加一下

Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0134/index.md]: ok, a new pr #13212 is opened to resolve this issue.

**Pull request merged**